src/components/EarnDashboard/components/ProviderDetails/WalletStakingByAsset.tsx

```diff
@@ -42,7 +43,11 @@ export const WalletStakingByAsset: React.FC<StakingPositionsByAssetProps> = ({ i
   const stakingOpportunities = useAppSelector(
     selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
   )
-  const filteredDown = stakingOpportunities.filter(e => ids.includes(e.id as OpportunityId))
+
+  useGetZapperAppsBalancesOutputQuery()
+  const filteredDown = stakingOpportunities
+    // @ts-ignore FIXME
```

Comment:
are we fixable?

Reply:
d1ad870ca301e83c11b8ba0ed54110be2fbcc26e